ucsb-cs156-f24/proj-happycows-f24-10
# Fix bug with wealth sorting mishandling commas

## Overview

In this PR, the table should now handle sorting by total wealth properly. Previously, the table would place values less than $1,000 higher than values greater than $1,000. In addition, a test is added that checks whether sorting is done correctly.

## Screenshots

<img width="806" alt="image" src="https://github.com/user-attachments/assets/a42235c7-fb57-4a7b-afc0-0771d108f7f7">
<img width="800" alt="image" src="https://github.com/user-attachments/assets/6681a76e-b052-43b9-a63f-09c8442137ce">
Previously, the orders would have been $1,101.10, $10,000.00, $198.73 and $198.73, $10,000.00, $1,101.10 respectively.


## Feedback Request

Please give suggestions on the following:
- The accessor is now just the raw value instead of the formatted one, I don't think that should break anything but I could've missed something.

## Validation

1. Download this branch.
2. Run the project.
3. Visit a commons buy enough cows for a user to have <$1,000 total wealth. 
4. Check the leaderboard. 
5. Sort by wealth in the table, and verify that the users are in the correct order. You may need to login multiple times as different users to have multiple users show up in the table.

Alternatively:
1. Check my dokku deploy here: https://happycows-dev-justinwu13.dokku-10.cs.ucsb.edu/
2. Visit the test commons.
3. Open the leaderboard, sort by wealth in the table, and verify that the users are in the correct order.

## Tests


## Linked Issues

Closes #7
